core/computed/metrics/lantern-metric.js

```diff
@@ -11,6 +11,24 @@ import {ProcessedNavigation} from '../processed-navigation.js';
 
 /** @typedef {import('../../lib/lantern/metric.js').Extras} Extras */
 
+/**
+ * @param {LH.Artifacts.MetricComputationDataInput} data
+ * @param {LH.Artifacts.ComputedContext} context
+ */
+async function getComputationDataParams(data, context) {
+  // TODO: remove this fallback when lighthouse-pub-ads plugin can update.
+  const gatherContext = data.gatherContext || {gatherMode: 'navigation'};
+  if (gatherContext.gatherMode !== 'navigation') {
+    throw new Error(`Lantern metrics can only be computed on navigations`);
+  }
+
+  const graph = await PageDependencyGraph.request(data, context);
+  const processedNavigation = await ProcessedNavigation.request(data.trace, context);
+  const simulator = data.simulator || (await LoadSimulator.request(data, context));
+
+  return {simulator, graph, processedNavigation};
+}
+
 class LanternMetric extends Metric {
```

Comment:
Is this still used anywhere?

Reply:
> I overlooked lantern-ttb

Does that mean leaving this here is intentional?

> btw why isnt that in predictive-perf (thats how I missed it)?

Unsure but seems like lantern TBT is computed differently from other metrics so it may not have made sense to include in lantern db tests.